backend/tracim_backend/models/tracim_session.py

```diff
@@ -0,0 +1,24 @@
+from contextlib import contextmanager
+
+from sqlalchemy.orm import Session
+
+
+class TracimSession(Session):
+    """
+    Subclass of Sqlalchemy session to add tracim specific stuff
+    """
+
+    def __init__(self, **kwargs):
+        Session.__init__(self, **kwargs)
+        self.allow_revision_deletion = False
+
+
+@contextmanager
+def unsafe_tracim_session(session: TracimSession):
```

Comment:
I would prefer a naming with "content_revision" (this context manager only allow "revision_deletion").

Reply:
renamed to "unprotected_content_revision"